plugins/source/aws/client/resolvers.go

```diff
@@ -138,7 +138,9 @@ func MarshaledJsonResolver(path string) schema.ColumnResolver {
 	return func(_ context.Context, meta schema.ClientMeta, r *schema.Resource, c schema.Column) error {
```

Comment:
Do we have a test for this function? if not I think we should add it and add the test case that causes this.

Reply:
This PR will not be merged. See @amanenk's PR... #2486